src/wasm-link.h

```diff
@@ -124,8 +125,17 @@ struct LinkerInputBinary {
   Index memory_page_offset;
 
   Index table_elem_count;
+  Index function_count;
```

Comment:
might be nice to make the member data private at some point, but this is fine for now.